Cannot install Properties Panel Extension Example

__Describe the Bug__
Executing `npm install` for the [Properties Panel Extension Example](https://github.com/bpmn-io/bpmn-js-examples/tree/master/properties-panel-extension) results in the following error:

> npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: bpmn-js-example-properties-panel-extension@0.0.0
npm ERR! Found: bpmn-js@8.2.0
npm ERR! node_modules/bpmn-js
npm ERR!   bpmn-js@"^8.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer bpmn-js@"^3.x || ^4.x || ^5.x || ^6.x || ^7.x" from bpmn-js-properties-panel@0.37.6
npm ERR! node_modules/bpmn-js-properties-panel
npm ERR!   bpmn-js-properties-panel@"^0.37.6" from the root project


__Steps to Reproduce__

<!-- Clear steps that allow us to reproduce the error. -->

1. Clone the repository.
2. Execute `npm install` in the directory properties-panel-extension.
3. The error is thrown.


__Expected Behavior__

The command should go through without errors, allowing me to run the example.


__Environment__

- Node version: v15.8.0
- NPM version: 7.5.4
- OS: Linux Mint 19.3
- Library version: 8.2.0

--- a/properties-panel-extension/package.json
+++ b/properties-panel-extension/package.json
@@ -27,7 +27,10 @@
   "license": "MIT",
   "devDependencies": {
     "babel-core": "^6.26.0",
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
     "babel-preset-env": "^1.6.1",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
     "babelify": "^8.0.0",
     "grunt": "~1.0.1",
     "grunt-browserify": "^5.0.0",
@@ -36,16 +39,13 @@
     "grunt-contrib-less": "^1.0.1",
     "grunt-contrib-watch": "~1.0.0",
     "load-grunt-tasks": "~3.5.0",
-    "stringify": "^3.1.0",
-    "babel-plugin-transform-flow-strip-types": "^6.22.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-stage-0": "^6.24.1"
+    "stringify": "^3.1.0"
   },
   "dependencies": {
     "bpmn-js": "^8.2.0",
-    "bpmn-js-properties-panel": "^0.37.6",
+    "bpmn-js-properties-panel": "^0.40.0",
     "camunda-bpmn-moddle": "^4.5.0",
-    "diagram-js": "^6.8.2",
+    "diagram-js": "^7.2.0",
     "jquery": "^3.5.1",
     "min-dash": "^3.5.2"
   }

